dotnet/test/common/BUILD.bazel

```diff
@@ -100,3 +103,39 @@ dotnet_nunit_test_suite(
         framework("nuget", "NUnit"),
     ],
 )
+
+dotnet_bidi_test_suite(
```

Comment:
It makes sense to create separate test targets for BiDi, but I think it might be easier for you to automatically handle this logic in `dotnet_nunit_test_suite` to automatically generate extra targets for BiDi and extend `env` based on the test file path.

Reply:
Is there a pattern of this I can follow? It's a little complicated because I want to figure out both:
1. Test suite runs all tests except what's in BiDi directory without BiDi Enabled
2. Test suite runs all tests in BiDi directory, plus specific tagged tests with BiDi enabled

Reply:
Nothing like that is implemented, if the current approach works, I suggest you stick to it and then we can improve.

Reply:
I haven't fixed the other things preventing bazel from working, yet. 😂